verible/verilog/analysis/descriptions.h

```diff
@@ -35,14 +35,16 @@ struct LintConfigParameterDescriptor {
   std::string description;
 };
 
-struct LintRuleDescriptor {
-  LintRuleId name;          // ID/name of the rule.
-  absl::string_view topic;  // section in style-guide
-  std::string desc;         // Detailed description.
-  std::vector<LintConfigParameterDescriptor> param;
-};
-
-}  // namespace analysis
-}  // namespace verilog
-
-#endif  // VERIBLE_VERILOG_ANALYSIS_DESCRIPTIONS_H_
+std::string format_long_description(const std::string& description) {
```

Comment:
Is this new function used anywhere?  If so, you may declare it in a .h file, but define it in the corresponding .cc file.